src/3d/framegraph/qgsambientocclusionrenderview.cpp

```diff
@@ -0,0 +1,183 @@
+/***************************************************************************
+  qgsambientocclusionrenderview.cpp
+  --------------------------------------
+  Date                 : June 2024
+  Copyright            : (C) 2024 by Benoit De Mezzo and (C) 2020 by Belgacem Nedjima
+  Email                : benoit dot de dot mezzo at oslandia dot com
+ ***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+
+#include "qgsambientocclusionrenderview.h"
+#include <Qt3DRender/QCamera>
+#include <Qt3DRender/QLayerFilter>
+#include <Qt3DRender/QLayer>
+#include <Qt3DRender/QRenderTargetSelector>
+#include <Qt3DRender/QRenderTarget>
+#include <Qt3DRender/QTexture>
+#include <Qt3DRender/QClearBuffers>
+#include <Qt3DRender/qsubtreeenabler.h>
```

Comment:
unneeded includes?

Reply:
`Qt3DRender/qsubtreeenabler.h` is needed for macos build